api-2.0/src/test/java/org/openmrs/module/attachments/AttachmentsServiceTest.java

```diff
@@ -144,7 +147,7 @@ public void getAttachments_shouldReturnAllAttachments() throws Exception {
 		
 		// attachments not bound to any visits/encounters
 		expectedAttachments.addAll(asAttachments(testHelper.saveComplexObs(null, 2, 0)));
-		
+		Collections.sort(expectedAttachments, Comparator.comparing(Attachment::getDateTime).reversed());
```

Comment:
Nice! This this solve the issue then?

Reply:
Yes , It solved the issue . Others methods will not create this error in my machine. But i think that adding this sorting method to all the other test cases will make sure that this issue is not occur again for any reason.